cmd/brew-alias.rb

```diff
@@ -71,6 +71,13 @@ def remove
       script.unlink
       symlink.unlink
     end
+
+
+    def edit
+      odie "Cannot edit 'brew-#{name}': Does not exist" if which("brew-#{name}").nil?
```

Comment:
In testing it out after the fact, I also discovered this would be a really useful feature. Was just waiting to make sure this was a direction you were ok with.